packages/bundle-size/src/utils/prepareFixture.js

```diff
@@ -24,6 +24,7 @@ module.exports = async function prepareFixture(fixture) {
     ast: false,
     code: true,
 
+    configFile: false,
     babelrc: false,
```

Comment:
```suggestion
    // This instance of Babel should ignore all user's configs and apply only our plugin
    configFile: false,
    babelrc: false,

```

Can we add a comment?